src/test/java/com/github/iveina2021/automationexercises/SignUpPage.java

```diff
@@ -0,0 +1,132 @@
+package com.github.iveina2021.automationexercises;
+
+// page_url =  https://automationexercise.com/signup/
+
+import com.codeborne.selenide.SelenideElement;
+import org.openqa.selenium.support.FindBy;
+
+import static com.codeborne.selenide.Selenide.page;
+
+public class SignUpPage {
+    @FindBy(css = ".login-form > .title")
+    public SelenideElement enterAccountInformationLabel;
+
+    @FindBy(css = "input[value='Mr']")
+    public SelenideElement genderCheckboxMr;
+
+    @FindBy(css = "input[id='password']")
+    public SelenideElement passwordInput;
+
+    @FindBy(css = "select[id='days']")
+    public SelenideElement daysDropDown;
+
+    @FindBy(css = "select[id='days'] option[value='2']")
+    public SelenideElement getDay2DropDown;
+
+    @FindBy(css = "select[id='months']")
+    public SelenideElement monthsDropDown;
+
+    @FindBy(css = "select[id='months'] option[value='4']")
+    public SelenideElement getMonthApril;
+
+    @FindBy(css = "select[id='years']")
+    public SelenideElement yearsDropDown;
+
+    @FindBy(css = "option[value='2019']")
+    public SelenideElement getYear2019;
+
+    @FindBy(css = "input[id='newsletter']")
+    public SelenideElement newsletterCheckbox;
+
+    @FindBy(css = "input[id='optin']")
+    public SelenideElement receiveSpecialOffersCheckbox;
+
+    @FindBy(css = "div[class='form-group'] label")
+    public SelenideElement dateBirthLabel;
+
+    @FindBy(css = "input[id='first_name']")
+    public SelenideElement firstNameInput;
+
+    @FindBy(css = "input[id='last_name']")
+    public SelenideElement lastNameInput;
+
+    @FindBy(css = "input[id='company']")
+    public SelenideElement companyInput;
+
+    @FindBy(css = "input[id='address1']")
+    public SelenideElement addressInput;
+
+    @FindBy(css = "input[id='address2']")
+    public SelenideElement addressTwoInput;
+
+    @FindBy(css = "select[id='country']")
+    public SelenideElement selectCountryDropDown;
+
+    @FindBy(css = "option[value='Canada']")
+    public SelenideElement getCountryCanada;
+
+    @FindBy(css = "input[id='state']")
+    public SelenideElement stateInput;
+
+    @FindBy(css = "input[id='city']")
+    public SelenideElement cityInput;
+
+    @FindBy(css = "input[id='zipcode']")
+    public SelenideElement zipcodeInput;
+
+    @FindBy(css = "input[id^='mobile']")
```

Comment:
**question** if you look at CSS selectors 
https://www.w3schools.com/csSref/css_selectors.php

You may see that this '^' means 'startsWith',

i.e. in your scenario id attribute should start with 'mobile'.

Is it really your scenario? Can we omit this ^ and use matching by equality.


Example:
[attribute^=value] | a[href^="https"] | Selects every <a> element whose href attribute value begins with "https"
-- | -- | --

Reply:
Fixed it